### ISSUE TITLE
JAWS does not work on Windows

### ISSUE THREAD

**Issue by dpurrington** [serverless/serverless#235]
I performed all the steps in the quick start but got this error:
"errorMessage":"Cannot find module 'aws_modules\\greetings\\hello\\handler'","errorType":"Error","stackTrace":["Function.Module._resolveFilename (module.js:338:15)","Function.Module._load (module.js:280:25)","Module.require (module.js:364:17)","require (module.js:380:17)"]}

I had made no manual edits to my generated files, and the zip file contained the handler.js at the expected location.

I was able to reproduce this by testing the Lambda function itself. After speaking with some folks on glitter, I tried running "jaws run" from within the aws_modules/greetings/hello folder. It returned this error: Error: Cannot find module 'D:\jaws-hello-EywHBGFgl\aws_modules\greetings\hello/aws_modules\greetings
\hello\handler.js'. Note the duplication of "aws_modules\greetings\hello". I suspect something is calculating CWD or relative paths and getting mixed up.

I tried this in PowerShell, Git bash, and cmd, and all produced the same result.

**Comment by sixlettervariables** [comment]
Dupe of #217 fixed in PR #230 if you'd like to give it a try.

**Comment by sixlettervariables** [comment]
Err #219.

**Comment by dpurrington** [comment]
I've got the PR checked out. How do I test it?

**Comment by sixlettervariables** [comment]
The easiest way is to `npm install` in that directory (to get all of the deps loaded), and then instead of calling `jaws` without a path specifier in your project, you can point to the `/path/to/bin/jaws` from that PR.

**Comment by dpurrington** [comment]
I think I'm the wrong person to test this because I don't know enough about
npm. I'm missing dependencies. I tried npm-install-missing and it failed
with an exception.

I'm sure node/npm can be made to work on Windows, but learning it while
debugging problems in the ecosystem is too hard. I'm going to switch over
to a Vagrant/Linux box so I can be more useful. Thanks!

**Comment by RLC85** [comment]
For those that stumble on this problem this issue is cause due to the wrong directory separators in `awsm.json`

`cloudformation.Handler`

**Comment by dzolnjan** [comment]
I'm getting exactly same error on Windows.

@RLC85 could you please be more specific about how fix this?

**Comment by eahefnawy** [comment]
Now that Serverless is released, is anyone still facing this issue? :)

**Comment by zanon-io** [comment]
@eahefnawy, yes, I do. I'm getting the same issue running on Windows after following the [documentation](http://docs.serverless.com/v0/docs) and running `serverless function run`.

> Error: Cannot find module 'C:\Users\Zanon\Documents\GitHub\aws-serverless-demo\serverless\serverlessVyhGVK8g\back\modules\greetings\hello\modules\greetings\hello\handler.js'

The problem is the duplication of **\modules\greetings\hello**.

I've also tried with the master branch (since the NPM version seems to be delayed by a few days) and changing a few paths (using POSIX compatibility) as was done in this [PR #218](https://github.com/serverless/serverless/pull/218) but I couldn't make it to work.

Is there any workaround until this is fixed? (running in a Linux machine is acceptable, if there is none available).

Thanks.

**Comment by ddausch** [comment]
Did anyone get this resolved in a Windows env? I am using serverless v.0.0.15. I opened this issue:
https://github.com/serverless/serverless/issues/444 a few days ago and was referred here but do not see any solutions listed.

**Comment by joostfarla** [comment]
Windows support is fixed in https://github.com/serverless/serverless/pull/457.
